datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -1576,6 +1576,26 @@ type DatasetProperties {
     External URL associated with the Dataset
     """
     externalUrl: String
+
+    """
+    Created date associated with the Dataset
```

Comment:
Date is a little confusing. It will be timestamp millis

Reply:
done